### DISCUSSION TITLE
Does otel-webserver-module support nginx-quic version?

### DISCUSSION THREAD

**Discussion by FanL1u** [open-telemetry/opentelemetry-cpp-contrib#239] [Q&A]
Hello, I tried opentelemetry webserver on nginx 1.23.1 mainline. It works well. But it didn't work on the nginx-quic version. Does it support nginx-quic or I made some mistakes when using it?

**Comment by DebajitDas** [comment]
Hi @Claireecf,
As of now, we have not provided support for nginx-quic.

**Reply by FanL1u** [reply]
Thanks for your reply. Do you plan to support it?
